src/synthesizer/parametric/sf_hist.py

```diff
@@ -383,147 +375,59 @@ def _sfr(self, age):
         return 0.0
 
 
-class ExponentiallyDeclining(Common):
```

Comment:
We still do need these

Reply:
Isnt this the exponential form: np.exp(-(self.max_age-age) / self.tau)

Reply:
I checked, this is equivalent to what I've done but swapping the sign of `\tau`. We could implement it that way or simply swap the sign of `\tau` if that allows us to align with other conventions?

Reply:
I'd keep both. It's nice sign posting. Makes it very clear to the user what to use if they want a certain thing. Multi-use classes/functions just add to confusion. 

By all means have one inherit from the other and swap the sign on `tau` to reduce boilerplate but we 100% should leave these classes available.

Reply:
I disagree, I think a single model makes more sense. We just need to decide on our `\tau` convention and be clear. Why have two models when one will suffice? Especially since you might like to explore a range of \tau encompassing positive and negative values.

Another part of the issue is that "Declining" and "Increasing" are a little ambiguous. In common usage a "DecliningExponential" means a galaxy whose recent star formation is less than it previously was. However, since we use age instead of time as our parameter a "DecliningExponential" is actually increasing SFR with age.

Reply:
I'm not going to capitulate on this one specifically. Keep both.

> Why have two models when one will suffice? 
Because it is immediately obvious to the user without them needing to think about what conventions we chose. They just pass a number. 

> Especially since you might like to explore a range of \tau encompassing positive and negative values.
That can be done with the base model regardless.

I agree it's pointless having all the redundant definitions if they can all just stem off a base class but the labelling alone makes it worthwhile.

> Another part of the issue is that "Declining" and "Increasing" are a little ambiguous. In common usage a "DecliningExponential" means a galaxy whose recent star formation is less than it previously was. However, since we use age instead of time as our parameter a "DecliningExponential" is actually increasing SFR with age.
That strikes me as a big problem in itself. We should follow what you might expect (as you describe). With named classes that just take a `tau` that's easily done with the specific naming because the user just passes a `tau` and should get a declining SFH (declining from a peak in the past). 
If you instead use a one size fits all method they need to know our convention, know it is backwards and then pass the correct `tau` following our choices.

Reply:
I have harmonised tau with the standard definition where tau is positive for a declining exponential. I've also added a DecliningExponential where tau is forced to be positive.